locales/en-GB/app.json

```
@@ -150,6 +150,9 @@
       "effect_volume_description": "Adjust the volume at which reactions and hand raised effects play",
       "effect_volume_label": "Sound effect volume"
     },
+    "background_blur_header": "Background",
+    "background_blur_label": "Blur the background of the video",
+    "blur_not_supported_by_browser": "(Background blur is not supported by this browser)",
```

```suggestion
    "blur_not_supported_by_browser": "(Background blur is not supported by this device)",
```

...because this message also appears when EC is embedded as widget and the user isn't aware that it is a browser.